examples/mouse_picking.rs

```diff
@@ -57,16 +60,6 @@ fn setup(
     commands
         .spawn(Camera3dBundle {
             transform: Transform::from_xyz(-2.0, 2.0, 2.0).looking_at(Vec3::ZERO, Vec3::Y),
-            camera: Camera {
-                // Define a viewport so we can verify screenspace rays are being constructed to
-                // account for viewport size.
-                viewport: Some(bevy::render::camera::Viewport {
-                    physical_position: UVec2::new(200, 200),
-                    physical_size: UVec2::new(400, 400),
-                    ..default()
-                }),
-                ..default()
-            },

```

Comment:
If this code does not work, that means viewport support is broken and needs to be fixed.